cax/config.py

```diff
@@ -178,37 +183,58 @@ def processing_script(host):
 #SBATCH --qos=xenon1t
 #SBATCH --partition=xenon1t
 #SBATCH --mail-user=pdeperio@astro.columbia.edu
-
 export PATH=/project/lgrandi/anaconda3/bin:$PATH
+export PROCESSING_DIR=/project/lgrandi/xenon1t/processing/{name}_{pax_version}
         """
     elif host == "tegner-login-1": # Stockolm-specific script options
-        script_template = """
-#SBATCH --output=/cfs/klemming/projects/xenon/common/xenon1t/processing/logs/{name}_{pax_version}_%J.log
-#SBATCH --error=/cfs/klemming/projects/xenon/common/xenon1t/processing/logs/{name}_{pax_version}_%J.log
+        script_template += """
+#SBATCH --output=/cfs/klemming/projects/xenon/common/xenon1t/processing/logs/{name}_{pax_version}.log
+#SBATCH --error=/cfs/klemming/projects/xenon/common/xenon1t/processing/logs/{name}_{pax_version}.log
 #SBATCH --account=xenon
 #SBATCH --partition=main
 #SBATCH -t 72:00:00
 #SBATCH --mail-user=Boris.Bauermeister@fysik.su.se
 
-source /afs/pdc.kth.se/home/b/bobau/load_4.8.4.sh
+#Load some basic modules from the PDC
+module add heimdal/standard
+module add gcc/4.8.4
+module add openmpi/1.8-gcc-4.8
+module add gsl/1.16
+module add fftw/3.3.4-gcc-8.4-openmpi-1.8-double
+
+export PROCESSING_DIR=/cfs/klemming/projects/xenon/xenon1t/processing/{name}_{pax_version}  
         """
     else:
         raise NotImplementedError("Host %s processing not implemented",
                                   host)
 
     # Script parts common to all sites
     script_template += """
-export PROCESSING_DIR={processing_dir}
 mkdir -p ${{PROCESSING_DIR}} {out_location}
 cd ${{PROCESSING_DIR}}
 rm -f pax_event_class*
-
 source activate pax_{pax_version}
+        """
+    
+    if host == "midway-login1":
+      script_template += """
+echo time cax-process --name {name} --in-location {in_location} --host {host} --pax-version {pax_version} --pax-hash {pax_hash} --out-location {out_location} --cpus {ncpus}
+time cax-process --name {name} --in-location {in_location} --host {host} --pax-version {pax_version} --pax-hash {pax_hash} --out-location {out_location} --cpus {ncpus}
+        """
+    elif host == "tegner-login-1": # Stockolm-specific script options
+        script_template += """
+echo time cax-process --name {name} --in-location {in_location} --host {host} --pax-version {pax_version} --pax-hash {pax_hash} --out-location {out_location} --cpus {ncpus}
+srun time cax-process --name {name} --in-location {in_location} --host {host} --pax-version {pax_version} --pax-hash {pax_hash} --out-location {out_location} --cpus {ncpus}        
+        """
```

Comment:
Can we just make `cax-process` and `srun` variables so we don't need the IF here?